platforms/core-configuration/object-graph-serialization/src/main/kotlin/org/gradle/configurationcache/serialization/Combinators.kt

```diff
@@ -18,71 +18,24 @@ package org.gradle.configurationcache.serialization
 
 import org.gradle.configurationcache.extensions.uncheckedCast
 import org.gradle.configurationcache.extensions.useToRun
-import org.gradle.configurationcache.problems.DocumentationSection
-import org.gradle.configurationcache.problems.StructuredMessageBuilder
-import org.gradle.internal.classpath.ClassPath
-import org.gradle.internal.classpath.DefaultClassPath
-import org.gradle.internal.classpath.TransformedClassPath
 import org.gradle.internal.serialize.BaseSerializerFactory
 import org.gradle.internal.serialize.Decoder
 import org.gradle.internal.serialize.Encoder
 import org.gradle.internal.serialize.Serializer
-
 import java.io.File
 import java.io.ObjectInputStream
 import java.io.ObjectOutputStream
-
 import kotlin.coroutines.Continuation
 import kotlin.coroutines.CoroutineContext
 import kotlin.coroutines.coroutineContext
 import kotlin.coroutines.startCoroutine
 import kotlin.coroutines.suspendCoroutine
 
 
-internal
 fun <T> singleton(value: T): Codec<T> =
     SingletonCodec(value)
 
 
```

Comment:
functions were moved to platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/Unsupported.kt